
<launch>
<!-- JOYSTICK nodes -->
  <node pkg="joy" type="joy_node" name="joy_node_right" >
    <remap from="joy" to="joy_right" />
    <rosparam command="load"
              file="$(find excavator)/launch/joy_param_right.yaml" />
  </node>
  <node pkg="joy" type="joy_node" name="joy_node_left" >
    <remap from="joy" to="joy_left" />
    <rosparam command="load"
              file="$(find excavator)/launch/joy_param_left.yaml" />
  </node>

</launch>